Dockerfile

```diff
@@ -150,10 +152,12 @@ ENV PATH /home/docs/.pyenv/shims:$PATH:/home/docs/.pyenv/bin
 RUN pyenv install $PYTHON_VERSION_27 && \
     pyenv install $PYTHON_VERSION_37 && \
     pyenv install $PYTHON_VERSION_36 && \
+    pyenv install $PY_VERSION_35 && \
```

Comment:
At least this typo broke the current master.

Reply:
Thanks. Fixed at #96